### ISSUE TITLE
List astropy coordinated affiliated packages on main documentation page

### ISSUE THREAD

**Issue by adrn** [astropy/astropy#8221]
As per discussion at astropy winter meeting 2018, we should add a list of the astropy-coordinated affiliated packages to the main docs page.

**Comment by pllim** [comment]
How about a webring? 🤔

**Comment by bsipocz** [comment]
this is more like an issue for the astropy website, isn't it?

**Comment by mhvk** [comment]
Looking at the main page, it is not obvious how to do this. However, an easy change would be to move the astropy-coordinates packages to a separate section on the affiliated package page, and then change the lead sentence from
```
The Astropy Project is a community effort to develop a common core package for Astronomy in Python and foster an ecosystem of interoperable astronomy packages.
```
to
```
The Astropy Project is a community effort to develop a common core package for Astronomy in Python and foster an ecosystem of interoperable astronomy packages, both within and outside of the project.
```
(with links, of course).

p.s. It is really for the web site, but the advantage of raising the issue here is that more people will see it... Though I guess one could pass things by astropy-dev as well.

**Comment by pllim** [comment]
@bsipocz , maybe the desire is to move that info from Astropy website to the docs here? @adrn , can you please clarify? Thanks.

**Comment by adrn** [comment]
We had a discussion about improving visibility of the astropy-coordinated affiliated packages (e.g., `reproject`). My understanding of the conclusion from that discussion was that we should add, e.g., a new section to the main `index.rst` called "Astropy-coordinated affiliated packages" with links to: reproject, astroplan, regions, astroquery, specutils, ccdproc, photutils and a link to astropy.org/affiliated. That's all I meant! This just a way to improve cross-referencing concepts and to hack Google.

**Comment by adrn** [comment]
And @bsipocz - no, I don't mean the main website. I mean the documentation!

**Comment by mhvk** [comment]
Ah, yes, that is a good idea. Indeed, just a few days ago I tried to find a link to the affiliated packages on the main docs page...

**Comment by bsipocz** [comment]
just last year at the the coordination meeting we started to move away from duplicating information between docs.astropy.org that points to the docs of the core package and astropy.org that's the landing page of the package. 
I'm worried that with this we're moving back to the previous setup.
(That being said, my interpretation of the discussion that seeded this issue was  more about linking to the relevant coordinated package from wherever it's relevant, e.g. link to astroplan from appropriate coordinate pages, to reproject from parts of the visualization/wcs pages)

**Comment by pllim** [comment]
👎 on duplication as well. High risk of them being out of sync and add to even more confusion. If you move that info to the docs here, you will change the affiliated webpage to link to the docs?

**Comment by adrn** [comment]
From my perspective, the coordinated affiliated packages don't change on a timescale that we need to worry about this and it is far more important to increase visibility. All I'm suggesting is a list of links on the main docs page - no other changes.

**Comment by adrn** [comment]
(BTW: I'm not saying we can't do *more* to improve visibility, but the suggestion is minimal in scope to make it easy to implement!)

**Comment by astrofrog** [comment]
I agree with @adrn that we are not adding/removing coordinated packages fast enough to be worried about duplication.

**Comment by pllim** [comment]
Okie dokie then. At least make it clear in the dev doc that if one add a package to website, one should also list it on the page here, and vice versa.

**Comment by bsipocz** [comment]
I just point back to last year's meeting when we decided to do the exact opposite and remove the affiliated package related pages from core as well as put the contributing docs in one place as it was said to be confusing to have it at multiple location. 
@kelle?

(I'm not worried about the out of sync stuff, the page could even get the data from the same json file for both location, I really just want to avoid this yearly back and forth of restructuring).

**Comment by olebole** [comment]
One (completely?) other and specific aspect to improve visibility: In Debian (and maybe in Fedora as well) one could add the coordinated packages to the list of packages "suggested" by astropy, which would bring them up in the install program (apt, or a graphical installer). For this, it would be nice if the astropy `CHANGES.rst` could have an entry "added pyvo to the list of coordinated packages", or if it would at least reflected by an attribute in `affiliated.json`. This would somehow allow to (manually) update the suggestions. Or any other way to trigger me (and maybe other packagers).

**Comment by bsipocz** [comment]
I totally support adding an attribute in `affiliated.json`, maybe even keep a separate changelog for the website or such to add notes for changes in the affiliated/coordinated list, but still feel it doesn't belong to the core package docs or changelog.

**Comment by pllim** [comment]
I think this came up again in Astropy Coordination Meeting 2025. @taldcroft showed a proof-of-concept from his branch during a breakout session but I don't see any PR open from it, hmm.

<img width="1063" height="1104" alt="Image" src="https://github.com/user-attachments/assets/8c3a1c5f-e7ed-40c5-9cc8-b863fd57e532" />

**Comment by taldcroft** [comment]
This is still on my high-priority to-do list, but I haven't had time to get back to it since the meeting. I've assigned myself to the issue.

**Comment by astrofrog** [comment]
For anyone interested in working on this (cc @taldcroft) I think the sunpy docs have done a great job at this (they also have other packages than the core sunpy package): https://sunpy.org

We might even want to consider adapting what they have done rather than reinvent the wheel

**Comment by astrofrog** [comment]
In particular:

<img width="1245" height="694" alt="Image" src="https://github.com/user-attachments/assets/4073ed15-2491-409b-ac5f-912907d4ec9d" />

**Comment by taldcroft** [comment]
@astrofrog - I like that a lot for astropy.org. I especially like the theme, it looks really great - clean, modern, inviting, and sunny! We should adapt that for astropy.

The project I'm working on now is a bit distinct, namely improving visibility of coordinated packages from docs.astropy.org, since many people might land on that domain and never see astropy.org.

**Comment by Cadair** [comment]
I would encourage you to use the sunpy theme, it's a reasonably lightweight fork of pydata-sphinx-theme with the main changes to move all the sphinx navigation into the left sidebar and out of the top nav so that the top nav can be used for org-wide navigation. There's a few other things in sunpy-sphinx-theme you will probably want to disable in the config (such as our [stats](https://sunpy.goatcounter.com/). We are also working on cross-project search: https://github.com/sunpy/sunpy-sphinx-theme/pull/287 backed by the RTD API.

Obviously the sunpy theme has all the defaults for sunpy, but we want it to be generally useful as well, so if you find anything you can't configure out then let us know.

repo: https://github.com/sunpy/sunpy-sphinx-theme

**Comment by Cadair** [comment]
oh, the other thing you will want to do is move astropy.org to be a sphinx site, so that you can use the same theme to get the consistent experience.

**Comment by taldcroft** [comment]
> I would encourage you to use the sunpy theme...
> ... move astropy.org to be a sphinx site

@Cadair - totally agree. Anyone at Aperio interested in doing this as funded work on the `Core/coordinated Package Maintenance and Development` contract?

**Comment by pllim** [comment]
As said in Astropy Coordination Meeting 2025, I am hesitant for Astropy as a project to adopt a theme customized and named for another project, Sunpy. If we ever need anything from that theme, we will never have priority because it is Sunpy's theme, not ours. At least with pydata, it is generic and used by multiple projects (at the time of writing). I would view it differently were it somewhere more upstream like OpenAstronomy.

**Comment by Cadair** [comment]
I mean sunpy uses lots of astropy stuff, both infrastructure and otherwise and we are always given a lot of support when we need to change stuff, I see no reason why this wouldn't be true the other way around.

If someone wants to spin out the sunpy default config from sunpy-sphinx-theme then they are welcome to but I don't think anyone on the sunpy side is going to volunteer to do that when we have already made everything sunpy specific customisable.


> Anyone at Aperio interested in doing this as funded work on the `Core/coordinated Package Maintenance and Development` contract?

I think we are out of hours on that FR now.

**Comment by taldcroft** [comment]
OK, I was thinking that the "Sunpy theme" was just a light variant of pydata-sphinx-theme, but it looks to be rather more involved (although it does directly inherit from pydata-sphinx-theme). The goals of `sunpy-sphinx-theme` (below) align well with goals from the coordination meeting:
- To provide a consistent look and feel over all the sunpy.org [astropy.org] websites and subdomains.
- To give all documentation pages maintained by the SunPy [Astropy] project a branded appearance.
- To provide a global navigation bar over all these sites so that it is possible to navigate between the sunpy.org [astropy.org] site and all the documentation.

Kudos to SunPy for getting this done. Doing this for Astropy would be a substantial project, much bigger than I was considering in the near term.

**Comment by astrofrog** [comment]
@taldcroft it might not be that much effort though since the sunpy theme is configurable and could be used as is as an astropy theme with different config. Sunpy have already done all the hard work.

**Comment by bsipocz** [comment]
I vaguely recall that the sunpy approach was brought up as a good solution in the past already but was dismissed in favor of the "we write our search engine instead". 
Glad to see we got back on track with the more maintainable, and in the right spirit of collaborative open source option. So 💯 of reusing what is already available.
(and also 💯 for sending money towards Tom and Stuart as they consistently attacking actual issues/enhancements/bugs the project faces for a long time rather than generating new ones that have zero effect on users).
